assets/src/scripts/components/hydrograph/audible.js

```diff
@@ -0,0 +1,119 @@
+const { scaleLinear } = require('d3-scale');
+const { select } = require('d3-selection');
+const memoize = require('fast-memoize');
+const { createSelector, createStructuredSelector } = require('reselect');
+
+const { yScaleSelector } = require('./scales');
+const { tsDatumSelector } = require('./tooltip');
+
+const { dispatch, link } = require('../../lib/redux');
+const { Actions } = require('../../store');
+
+
+// Higher tones get lower volume
+const volumeScale = scaleLinear().range([2, .3]);
+
+const AudioContext = window.AudioContext || window.webkitAudioContext;
+const getAudioContext = memoize(function () {
+    return new AudioContext();
+});
+
+export const createSound = memoize(/* eslint-disable no-unused-vars */ tsKey => {
+    const audioCtx = getAudioContext();
```

Comment:
Hmmmmm... good question. I opted to use the single context rather than creating one for each tone so I didn't need to deal with managing a pool of them. I also saw some references online to browser errors if too many contexts are created at once. However, I didn't consider closing the context... it appears if we do that, we can't attach new nodes to it. So my feeling is to keep it open, but perhaps when changes are made to play more than just the two tones (first current and first compare series) there should be a pool of contexts that are closed after they are created.

Reply:
Sounds good.